index.html

```diff
@@ -0,0 +1,395 @@
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mwkakrqj" method="POST">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <div class="name-form">
+            <input
+              id="name"
+              placeholder="First name"
+            />
+            <input
+              id="surname"
+              placeholder="Last name"
+            />
+          </div>
+          <input
+            placeholder="Email address"
+          />
+          <textarea
+            name="message"
+            placeholder="Enter text here"
+          ></textarea>
+          <button type="submit" class="link centered-link">Send</button>
```

Comment:
💹 - [ ] Good job implementing the submit button correctly, the functionality is good but I think the UI needs some improvements.
- - Figma design
 
![Screenshot 2023-03-17 002839](https://user-images.githubusercontent.com/107632780/225774681-f59b9096-9f42-4bf1-99ae-8f3a0f2ede5f.png)

- - Your design
 
![Screenshot 2023-03-17 002918](https://user-images.githubusercontent.com/107632780/225774723-aa403db7-c4f8-43ca-9ed2-5825439b331a.png)

- [ ] According to the Figma design, the button should be centered in the form, Please consider following the Figma design carefully as the ability to replicate design is a great quality most combines look out for.  😄 😄 
- [ ] Also according to the design, the button text is `Get in touch, please consider updating your text to match the one in the design 😄 😄 
- [ ] Lastly, I think you have a black border around your button which is not in the design, Please consider removing the CSS property border property in order to remove it. 💯 🥇